### PR TITLE
fix: remove deprecated ioutil

### PR DESCRIPTION
**Pull request by fengshunli**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by sejust** [comment]
@fengshunli  It is recommended that you do not modify the files under the directory `depends/`. This are third-party library that needs to be updated according to they own version.
And Do not format comments that are not related to this submission.Code formatting can be mentioned another PR. Thanks.

**Comment by sejust** [comment]
Add  `Signed-off-by` in commit. See more at #1920

**Comment by fengshunli** [comment]
> Add `Signed-off-by` in commit. See more at #1920

done

**Comment by sejust** [comment]
fix the gofmt https://github.com/cubefs/cubefs/actions/runs/5242644622/jobs/9466305258?pr=2130

**Comment by fengshunli** [comment]
> fix the gofmt https://github.com/cubefs/cubefs/actions/runs/5242644622/jobs/9466305258?pr=2130

merge

File: blobstore/blobnode/core/disk/trash.go [hunk 1]
```diff
@@ -57,14 +56,15 @@ func (ds *DiskStorage) cleanTrash(ctx context.Context) (err error) {
...
 	trashProtection := time.Duration(ds.Conf.DiskTrashProtectionM) * time.Minute
 
-	fis, err := ioutil.ReadDir(trashPath)
+	fis, err := os.ReadDir(trashPath)
 	if err != nil {
 		span.Debugf("failed readdir, err:%v", err)
 		return
 	}
 	for _, fi := range fis {
+		info, _ := fi.Info()
```
Comment (sejust) [inline blobstore/blobnode/core/disk/trash.go:65]: the returned error cannot be ignored

File: metanode/util.go [hunk 2]
```diff
@@ -50,7 +50,8 @@ func sortDelExtFileInfo(files []os.FileInfo) []os.FileInfo {
 		}
 
 		if strings.HasPrefix(info.Name(), prefixDelExtent) {
-			newFiles = append(newFiles, info)
+			fileInfo, _ := info.Info()
```
Comment (sejust) [inline metanode/util.go:53]: the error cannot be ignored.

**Review by sejust — CHANGES_REQUESTED** [review]

File: metanode/util.go (hunk 2 above)
Comment (fengshunli) [reply metanode/util.go:53]: done

File: blobstore/blobnode/core/disk/trash.go (hunk 1 above)
Comment (fengshunli) [reply blobstore/blobnode/core/disk/trash.go:65]: done

**Review by sejust — APPROVED** [review]

File: blobstore/blobnode/base/util_test.go [hunk 3]
```diff
@@ -24,10 +23,11 @@ import (
...
 // Example:
-//   "Range": "bytes=0-99"
-//   "Range": "bytes=100-200"
-//   "Range": "bytes=-50"
-//   "Range": "bytes=150-"
+//
+//	"Range": "bytes=0-99"
+//	"Range": "bytes=100-200"
+//	"Range": "bytes=-50"
+//	"Range": "bytes=150-"
```
[the remark this answers is not in the document]
Comment (fengshunli) [reply blobstore/blobnode/base/util_test.go]: Commonly used indentation, here is two spaces

File: blockcache/bcache/manage.go [hunk 4]
```diff
@@ -543,53 +542,53 @@ func (d *DiskStore) checkBuildCacheDir(dir string) {
...
+//		}
+//		err = os.Rename(tmp, cachePath)
+//		if err != nil {
+//			log.LogErrorf("Rename block tmp file:%s err:%s!", tmp, err)
+//			return err
+//		}
+//		atomic.AddInt64(&d.usedSize, int64(len(data)))
+//		atomic.AddInt64(&d.usedCount, 1)
+//		log.LogDebugf("TRACE BCacheService flushKey Exit. key(%v) cachePath(%v)", key, cachePath)
+//		return nil
```
[the remark this answers is not in the document]
Comment (fengshunli) [reply blockcache/bcache/manage.go]: done

File: blobstore/blobnode/base/util_test.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (sejust) [reply blobstore/blobnode/base/util_test.go]: https://tip.golang.org/doc/comment

[the remark this answers is not in the document]
Comment (fengshunli) [reply blobstore/blobnode/base/util_test.go]: image
The format in the document is the same as my current submission

**Review by sejust — APPROVED** [review]

**Comment by leonrayang** [comment]
@fengshunli need combine the commits to several import one

**Comment by leonrayang** [comment]
@fengshunli the commits include some one not belong to you，need push the commits again.

**Comment by fengshunli** [comment]
> @fengshunli the commits include some one not belong to you，need push the commits again.

@leonrayang switch a branch push

**Pull request closed without merging**